src/pages/Checkout/index.js

```diff
@@ -25,66 +25,109 @@ import { connect } from "react-redux";
 
 import { cartTotal } from "../../utils/cartTotal";
 import { isMaxLength } from "../../utils/isMaxLength";
+import { setCartTotal } from "../../action/cart";
+import apiFetch from "../../utils/apiFetch";
 
-const Checkout = () => (
+const Checkout = ({ cart }) => (
   <div className={styles.wrapper}>
     <h2 className={styles.title}>Checkout</h2>
-    <ReduxFormWithList onSubmit={onSubmit} />
+    <ReduxFormWithList
+      onSubmit={async values => await onSubmit(values, cart)}
+    />
   </div>
 );
 
-const onSubmit = formData => {
-  console.log("formData", formData);
+const onSubmit = async (formData, cart) => {
+  const { cartList, cartTotal } = cart;
+  const product = [];
+  Object.values(cartList).map(({ qty, price, id }) => {
+    product.push({
+      quantity: qty,
+      productId: id,
+      pricePerOne: price,
+      totalPrice: price * qty
+    });
+  });
+  const bill = {
+    ...formData,
+    ...cartTotal,
+    delivery: DELIVERY,
+    product: product
+  };
+
+  const res = await apiFetch("/bills", {
+    method: "POST",
+    body: JSON.stringify(bill),
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json"
+    }
+  });
+
+  const data = res.json();
+
```

Comment:
after send data you need clear cartList

Reply:
and show success modal with text `Thank you for your order. Our manager will contact you`

and button `Ok` 
when you press `Ok` you navigate to home page